pyannote/audio/core/task.py

```diff
@@ -264,15 +330,324 @@ def prepare_data(self):
 
```

Comment:
I think this docstring needs to be updated. 
We can remove the "but for now, blah blah blah".